src/device/intrinsics/wmma.jl

```diff
@@ -10,113 +10,89 @@ using Core: LLVMPtr
 
 # Maps PTX types to Julia array types
 const map_ptx_to_jl_array = Dict(
```

Comment:
I'm not sure these whitespace changes are worth it.

Reply:
Oh, VS Code made them automatically. I'll try to undo them.